README.md

```diff
@@ -64,6 +64,30 @@ isRequired('My Field')('42'); // undefined, therefore assume valid
 isRequired({ message: 'Error' })(); // 'Error'
 ```
 
+Validation functions can optionally accept a second parameter including all of the current values.
+This allows comparing one value to another as part of validation.  For example:
+
+```js
+import { createValidator } from 'revalidate';
+
+const matchesValue = function(matchKey) {
```

Comment:
Can you change the function formatting here to match what you have in the source code for `matchesValue` in `src/validators/matchesValue.js` please?